### PR TITLE
Move templates to RHEL/CentOS/UBI 8

### PR DESCRIPTION
**Pull request by yselkowitz**
This has already been tested in image ecosystem:

https://github.com/openshift/cluster-samples-operator/pull/319

### CODE REVIEW

File: openshift/templates/rails-postgresql-persistent.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
             "from": {
               "kind": "ImageStreamTag",
               "namespace": "${NAMESPACE}",
-              "name": "ruby:2.5"
+              "name": "ruby:2.5-ubi8"
```
Comment (hhorak) [inline openshift/templates/rails-postgresql-persistent.json]: Version 2.5 is not latest anymore. We can either use `2.7-ubi7` or `2.6-ubi8`

File: openshift/templates/rails-postgresql-persistent.json [hunk 2]
```diff
@@ -368,7 +368,7 @@
               "from": {
                 "kind": "ImageStreamTag",
                 "namespace": "${NAMESPACE}",
-                "name": "postgresql:10"
+                "name": "postgresql:10-el8"
```
Comment (hhorak) [inline openshift/templates/rails-postgresql-persistent.json]: Here as well, we can use `12-el8` that is already available.

File: openshift/templates/rails-postgresql-persistent.json (hunk 1 above)
Comment (hhorak) [reply openshift/templates/rails-postgresql-persistent.json]: Hm, maybe 2.6 for now before https://github.com/sclorg/rails-ex/pull/130 is tested properly.

**Review by hhorak — CHANGES_REQUESTED** [review]
It seems to be worth updating the versions as well, otherwise LGTM.

**Comment by yselkowitz** [comment]
Changes incorporated, please re-review.

**Comment by hhorak** [comment]
Perfect, thanks.

**Review by hhorak — APPROVED** [review]

**Pull request merged**